### DISCUSSION TITLE
Epidemic trend analysis

### DISCUSSION THREAD

**Discussion by adamkucharski** [WHO-Collaboratory/collaboratory-epipipeline-community#22] [Ideas]
- Estimating epidemic growth rates taking into account overdispersion of cases and day-of-week variation in reporting
- R package to analyze time series of case counts

WHO Collaboratory Hackathon Idea.

**Comment by calthaus** [comment]
Just realized that [`trending`](https://github.com/reconverse/trending) is doing this already.
